src/plane_tools.rs

```diff
@@ -0,0 +1,168 @@
+use nalgebra::base::Vector3;
```

Comment:
TBH I don't see the added benefit of having all the `*_tools` modules. I would put this in a module called `plane`, together with the new `Plane` structure. Don't change anything now. I will do so in the followup cleanup.

Reply:
exactly